commerce-cloud-code/app_yourshophere/cartridge/experience/components/theme/skin.js

```diff
@@ -1,49 +1,43 @@
-'use strict';
-
-var Template = require('dw/util/Template');
-var PageRenderHelper = require('*/cartridge/experience/utilities/PageRenderHelper.js');
-var HashMap = require('dw/util/HashMap');
-var HashSet = require('dw/util/HashSet');
-
-var fontMap = {
+const HashSet = require('dw/util/HashSet');
+const partials = require('partials');
+const PageRenderHelper = require('*/cartridge/experience/utilities/PageRenderHelper.js');
+const fontMap = {
     'Salesforce Sans': 'components/fonts/salesforcesans',
```

Comment:
we dont need that mapping to a folder anymore, if we directly resolve it in the main partial

separate files could have their merits, as adding a font would just mean, we chnage the skin.json and upload a file (i.e. no need to change such a map here)


        if (font === 'components/fonts/nunito') {
            return '<link href="https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@600;700&display=swap" rel="stylesheet">';
        }

Reply:
I agree, I'll admit I put not too much thought into it as it was a quite large refactoring. We could export the fonts in JS files and require those. Some approach as before but without ISML

Reply:
yep - the isml approach had this useless map too. It's all good - just noticed.